assets/src/domain/eventEditor/services/apollo/queries/datetimes/useDatetimes.ts

```diff
@@ -1,12 +1,19 @@
+import { useMemo } from 'react';
+
 import useDatetimeQueryOptions from './useDatetimeQueryOptions';
 import { Datetime, DatetimesList } from '../../types';
 import useCacheQuery from '../useCacheQuery';
+import { getCacheIds } from '@sharedServices/predicates';
 
 const useDatetimes = (): Array<Datetime> => {
 	const options = useDatetimeQueryOptions();
 	const { data } = useCacheQuery<DatetimesList>(options);
 
-	return data?.espressoDatetimes?.nodes || [];
+	const nodes = data?.espressoDatetimes?.nodes || [];
+
+	const cacheIds = getCacheIds(nodes).join(':');
+
+	return useMemo(() => nodes, [cacheIds]);
```

Comment:
is is possible to have `nodes` being changed while `cacheIds` remains the same?

Reply:
No, `cacheId` is `uuid`.